Add /usr/lib/BiblioteQ search path to the launcher script.

--- a/biblioteq.sh
+++ b/biblioteq.sh
@@ -50,6 +50,17 @@
     exit $?
 fi
 
+if [ -f /usr/lib/BiblioteQ/BiblioteQ ] && \
+   [ -r /usr/lib/BiblioteQ/BiblioteQ ] && \
+   [ -x /usr/lib/BiblioteQ/BiblioteQ ]
+then
+    echo "Launching an official BiblioteQ (/usr/lib/BiblioteQ)."
+    set_qt_qpa_platformtheme "/usr/lib/BiblioteQ/BiblioteQ"
+    /usr/lib/BiblioteQ/BiblioteQ \
+	--configuration-file /etc/biblioteq.conf "$@"
+    exit $?
+fi
+
 if [ -f /usr/local/biblioteq/BiblioteQ ] && \
    [ -r /usr/local/biblioteq/BiblioteQ ] && \
    [ -x /usr/local/biblioteq/BiblioteQ ]
